spk/minio/src/wizard/install_uifile

```diff
@@ -0,0 +1,64 @@
+[
+	{
+		"step_title": "MinIO configuration",
+		"items": [
+		  {
+			"type": "textfield",
+			"subitems": [
+			  {
+				"key": "wizard_data_directory",
+				"desc": "Data directory location",
+				"defaultValue": "/volume1/minio",
+				"validator": {
+				  "allowBlank": false,
+				  "regex": {
+					"expr": "/^\\\/volume[0-9]{1,2}\\\/[^<>: */?\"]*/",
+					"errorText": "Path should begin with /volume?/ where ? is volume number (1-99)"
+				  }
+				}
+			  },
+			  {
+				"key": "wizard_access_key",
+				"desc": "MinIO access key",
+				"defaultValue": "minio-access-822791549661344",
+				"validator": {
+				  "allowBlank": false,
+				  "minLength": 3,
+				  "regex": {
+					"expr": "/^[^<>:*/?\"|]*$/",
+					"errorText": "Not allowed character in key"
+				  }
+				}
+			  },
+			  {
+				"key": "wizard_secret_key",
+				"desc": "MinIO secret key",
+				"defaultValue": "minio-secret-033309986404774",
```

Comment:
same proposal as for the access key:
I propose to use the simple default value "minio-123" as used in some minio examples.
Your default value looks like containing a random number but it does not! It will be the same for all package installations.

If the default values are as simple as minio/minio-123 it is obvious that one has to change it.

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/3689/commits/fb5b9e9813718d4052368d31117ea126ecc6b703